web_src/less/chroma/light.less

```
@@ -53,8 +53,8 @@
 .chroma .cm { color: #999988; } /* CommentMultiline */
 .chroma .c1 { color: #6a737d; } /* CommentSingle */
 .chroma .cs { color: #999999; } /* CommentSpecial */
-.chroma .cp { color: #999999; } /* CommentPreproc */
-.chroma .cpf { color: #999999; } /* CommentPreprocFile */
+.chroma .cp { color: #109295 } /* CommentPreproc */
+.chroma .cpf { color: #4C4DBC; } /* CommentPreprocFile */
```

```suggestion
.chroma .cpf { color: #4c4dbc; } /* CommentPreprocFile */
```
Other colors use lowercase.